templates/anndata_conversion.ipynb

```diff
@@ -0,0 +1,784 @@
+{
```

Comment:
<p>Maybe include the nice graphic from PR #1073 with the AnnData matrices structure.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/angelolab/ark-analysis/pull/1079/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>